slides/intro-03-what-makes-a-model.qmd

```diff
@@ -520,36 +523,55 @@ How do you use your new `tree_fit` model?
 
 ```{r tree-wflow-fit-2}
 tree_spec <-
-  decision_tree(cost_complexity = 0.002) %>% 
-  set_mode("classification")
+  decision_tree() %>% 
+  set_mode("censored regression")
 
 tree_fit <-
-  workflow(tip ~ ., tree_spec) %>% 
-  fit(data = taxi_train) 
+  workflow(event_time ~ ., tree_spec) %>% 
+  fit(data = cat_train) 
 ```
 
-## Your turn {transition="slide-in"}
+## Predict with your model `r hexes("parsnip", "workflows")`
 
-![](images/parsnip-flagger.jpg){.absolute top="0" right="0" width="150" height="150"}
+How do you use your new `tree_fit` model?
 
-*Run:*
+```{r tree-wflow-predict}
+predict(tree_fit, new_data = cat_test)
+```
 
-`predict(tree_fit, new_data = taxi_test)`
+## Predict with your model `r hexes("parsnip", "workflows")`
 
-*What do you get?*
+How do you use your new `tree_fit` model?
 
-```{r ex-predict-tree-fit}
-#| echo: false
-countdown::countdown(minutes = 3, id = "predict-tree-fit")
+```{r tree-wflow-predict-time}
+predict(tree_fit, new_data = cat_test, type = "time")
```

Comment:
In annotations, we should mention how we [reverse the sign](https://parsnip.tidymodels.org/reference/details_proportional_hazards_survival.html#linear-predictor-values) of the linear predictor for PH models. OR should this go on the slides?

Reply:
That's a good thing for the annotation #11